### ISSUE TITLE
[Keyboard Navigation - Adaptive Cards iOS - V1.1] "Adaptive cards Media” button is not accessible with swipe navigation on “Media.Sources” page.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6950]
https://microsoft.visualstudio.com/OS/_workitems/edit/29527290
